homeassistant/components/insteon_plm.py

```diff
@@ -6,115 +6,176 @@
 """
 import logging
 import asyncio
+import collections
 
 import voluptuous as vol
 
 from homeassistant.core import callback
-from homeassistant.const import (
-    CONF_PORT, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.const import (CONF_PORT, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.helpers.config_validation as cv
 from homeassistant.helpers import discovery
 
-REQUIREMENTS = ['insteonplm==0.7.5']
+REQUIREMENTS = ['insteonplm==0.8.2']
 
 _LOGGER = logging.getLogger(__name__)
 
 DOMAIN = 'insteon_plm'
 
 CONF_OVERRIDE = 'device_override'
+CONF_NEWNAMES = 'new_names'
 
 CONFIG_SCHEMA = vol.Schema({
     DOMAIN: vol.Schema({
         vol.Required(CONF_PORT): cv.string,
-        vol.Optional(CONF_OVERRIDE, default=[]): cv.ensure_list_csv,
-    })
+        vol.Optional(CONF_OVERRIDE, default=[]): vol.All(
+            cv.ensure_list_csv, vol.Length(min=1)),
+        vol.Optional(CONF_NEWNAMES, default=['no']): cv.string
```

Comment:
This is an invalid default. You want a string but default to a list with the entry 'no'.

Reply:
Also, if you want a boolean value, use `cv.boolean`.

Reply:
Done